all: resource optimization and netpol fixes (#1244)
elastisys/compliantkubernetes-apps: WIP-CHANGELOG.md

### Release notes

> - Several default resource requests and limits have changed. When upgrading these might need to be changed in some environments.

Reviewer: ```suggestion
- Several default resource requests and limits have changed. When upgrading these might need to be adjusted.
```

- Update the provider plugins to a supported version for the new Velero release
- Added support for anchors and aliases in override configs, not tested with merge aliases/tags
- Changed the default limit for diskLimit alert, from 66 to 75

> - Resources for multiple apps

Reviewer: ```suggestion
- Added some default resource requests and limits for multiple components
```
